.gitignore

```diff
@@ -1 +1,16 @@
+# OSX
```

Comment:
I think it's a reasonable idea, but I'm more inclined to follow the patterns in the bazel repo https://github.com/bazelbuild/bazel/blob/master/.gitignore since this lives in the bazelbuild org.